src/loaders/genome_collection/parse_tool_results.py

```diff
@@ -466,34 +501,42 @@ def _parse_heatmap_rows(
         for missing_trait_idx in missing_trait_idxs:
```

Comment:
Can you remind me when when this would happen?

Reply:
it's just our safety. But it should never happen for microtrait. In case of microtrait result, all genomes are having exactly same number of traits. It will happen when one genome has less number of traits comparing to other genomes. If you think this might create hidden bug for us, should we just raise here?

Reply:
I guess I would ask:
1) Document more or less what you said above in a comment by that code
2) The code should warn when there are missing traits

Reply:
:+1:

Reply:
👍